docs/index.rst

```diff
@@ -120,6 +120,46 @@ mac_address
 .. autofunction:: mac_address
 
 
+md5
+-----------
+
+.. module:: validators.md5
```

Comment:
Module on all of these should be ``validators.hashes`` (also change to file name form hashs.py to hashes.py)